lib/teracy-dev-core/processors/variables.rb

```diff
@@ -7,6 +7,13 @@ module Processors
     class Variables < TeracyDev::Processors::Processor
 
       def process(settings)
+        settings['variables'] = settings['variables'] || []
+        # Export extensions path to variables
+        settings['teracy-dev']['extensions'].each do |extension|
+          settings['variables']["#{extension['path']['extension']}-path"] = File.join(extension['path']['lookup'] || TeracyDev::DEFAULT_EXTENSION_LOOKUP_PATH,  extension['path']['extension'])
```

Comment:
extension_path is not extension name, need to get extension name from its manifest https://github.com/teracyhq/dev/blob/develop/lib/teracy-dev/util.rb#L36